templates/account.template.yaml

```diff
@@ -87,11 +87,7 @@ Resources:
               - Resource: '*'
                 Effect: Allow
                 Action:
-                    - codecommit:GetBranch
-                    - codecommit:GetCommit
-                    - codecommit:UploadArchive
-                    - codecommit:GetUploadArchiveStatus
-                    - codecommit:CancelUploadArchive
+                  - codestar-connections:UseConnection
```

Comment:
Not using codecommit anymore, but we need to ad some permissions for codestar-connections

Reply:
Add a comment regarding the connection of Github and codestar?